### DISCUSSION TITLE
Proposal to Implement temperature control

### DISCUSSION THREAD

**Discussion by brockar** [JaKooLit/Hyprland-Dots#480] [Ideas]
I would like to implement wl-gammarelay-rs, a tool similar to Redshift but for Wayland, as I find it crucial to have a temperature control feature for long working sessions on my PC.

To do this, wl-gammarelay-rs would need to be installed, either via the Arch AUR or by compiling it with Cargo. I will also take care of adding it to the configuration files.

My question is whether there is interest in having me add this feature. If so, should I include the module in all configurations, in just some of them, or not at all, leaving it only in the ModulesCustom section?

Looking forward to your feedback.

**Comment by dwilliam62** [comment]
Hyprland has a redshift feature now hyprsunset.  I haven't tried it yet

**Reply by brockar** [reply]
Well, I'll wait for Hyprland 0.45.0 to be available in Arch and try it. At the moment it is 0.44.1-1.
